project-base/storefront/components/Layout/OrderLayout/OrderLayout.tsx

```diff
@@ -25,18 +27,42 @@ export const OrderLayout: FC<OrderLayoutProps> = ({ activeStep, isTransportOrPay
                 <Header simpleHeader />
             </Webline>
             <Adverts positionName="header" withGapBottom withWebline />
+            {isLoading ? (
+                <OrderLayoutContentSkeleton>{children}</OrderLayoutContentSkeleton>
+            ) : (
+                <>
+                    <Webline>
+                        <OrderSteps activeStep={activeStep} domainUrl={url} />
+                    </Webline>
+                    <Webline>
+                        <div className="mb-24 flex w-full flex-col flex-wrap vl:mt-7 vl:mb-16 vl:flex-row">
+                            <div className="mb-16 w-full vl:mb-0 vl:min-h-[61vh] vl:flex-1 vl:pr-10">{children}</div>
+                            <div className="w-full vl:max-w-md">
+                                <OrderSummary isTransportOrPaymentLoading={isTransportOrPaymentLoading} />
+                            </div>
+                        </div>
+                    </Webline>
+                </>
+            )}
+            <Adverts positionName="footer" withGapBottom withWebline />
+        </>
+    );
+};
+
+const OrderLayoutContentSkeleton: FC = ({ children }) => {
+    return (
+        <>
             <Webline>
-                <OrderSteps activeStep={activeStep} domainUrl={url} />
+                <Skeleton className="h-10 w-full" />
             </Webline>
             <Webline>
                 <div className="mb-24 flex w-full flex-col flex-wrap vl:mt-7 vl:mb-16 vl:flex-row">
                     <div className="mb-16 w-full vl:mb-0 vl:min-h-[61vh] vl:flex-1 vl:pr-10">{children}</div>
-                    <div className="w-full vl:max-w-md">
-                        <OrderSummary isTransportOrPaymentLoading={isTransportOrPaymentLoading} />
+                    <div className="h-60 w-full vl:max-w-md">
+                        <Skeleton className="h-full w-full" />
                     </div>
                 </div>
             </Webline>
-            <Adverts positionName="footer" withGapBottom withWebline />
```

Comment:
Is this removal really wanted?

Reply:
Its not removed, git just doesn't understand me moving it:

<img width="830" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/91405fd7-7fc3-4586-97b4-93450c0123b0">